src/machine/machine_stm32_adc_f1.go

```diff
@@ -23,15 +23,6 @@ func InitADC() {
 	// Enable ADC clock
 	enableAltFuncClock(unsafe.Pointer(stm32.ADC1))
 
-	// set scan mode
```

Comment:
After the chip reset the ADC is already configured for "Single conversion mode". Scan mode is relevant when DMA is used.